CONTRIBUTING.md

```diff
@@ -11,6 +11,18 @@ expectations the Node Foundation requires of its contributors.
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Collaborator** is a subset of contributors who have been given write access to the repository.
 
+# Getting Involved with the Node.js Community Committee
```

Comment:
Looking at this holistically within the file, I think we should drop the title here, and make "Start Here" and "Teams and Working Groups" at the same level as "Logging Issues". As is, this title feels kinda redundant to me.